### DISCUSSION TITLE
How to apply SCI to real-world use case

### DISCUSSION THREAD

**Discussion by navveenb** [Green-Software-Foundation/sci#120] [General]
We have created a document on how to apply SCI for a real-world use case application.
Team, please review the document and add your feedback/questions via the comments section on the document
Link - https://docs.google.com/document/d/1tV1MSyJBtLlsAvTyS_voPxi_3D-ZmvIwcA79TB0z2Aw/edit?usp=sharing

**Comment by atg-abhishek** [comment]
Can you move this into the `wiki` instead - it will make it a lot more accessible for people to interact with it and maybe even make PRs, etc. against it?

**Reply by will-iamalpine** [reply]
@navveenb please confirm when it has a home on the wiki so we can reference it :)

**Comment by navveenb** [comment]
I missed this notification. I will ask @seanmcilroy29 on how we can move the document to wiki.

**Reply by navveenb** [reply]
Thanks @seanmcilroy29 for moving this to the wiki . The URL is - https://github.com/Green-Software-Foundation/software_carbon_intensity/wiki/Green-Software-Foundation-Reference-Guide-Manual
